src/main/java/spoon/reflect/visitor/ImportCleaner.java

```diff
@@ -201,6 +204,12 @@ void addImport(CtReference ref) {
 			}
 		}
 
+		private boolean isPackageImportedViaWildcardAndUnresolved(CtPackageReference packageReference) {
```

Comment:
Name is a bit weird to me; you can't import a package. Maybe something like this?

```suggestion
		private boolean isUsedInWildcardImport(CtPackageReference packageReference) {
```

Not crazy about that name either, but as a package itself can't be imported, I don't think we should... imply that it can.